pkg/relic/Makefile

```diff
@@ -3,11 +3,10 @@ PKG_URL=https://github.com/relic-toolkit/relic.git
 PKG_VERSION=0b0442a8218df8d309266923f2dd5b9ae3b318ce
 PKG_LICENSE=LGPL-2.1
 
-.PHONY: all ..cmake_version_supported
+include $(RIOTBASE)/pkg/pkg.mk
```

Comment:
Is there a reason for not applying the patch I proposed in https://gist.github.com/fjmolinas/3d77a86bfd6a87e4a1cfb46bb1dc863c ? To keep the old behavior `$(TOOLCHAIN_FILE): FORCE` must be used similar to the `ccn-lite` changes.

Reply:
I think it got lost somewhere. I developed on two machines, and not sure what happened. Will restore it.